set.mm

```diff
@@ -637993,6 +638049,158 @@ proof usage (see ~ mplelbas and ~ mpladd ).  (Contributed by SN,
       VCVVD $.
   $}
 
+  ${
+    $d ph x $.  $d F x $.  $d I x $.
+    fsuppssindlem1.z $e |- .0. = ( 0g ` G ) $.
```

Comment:
What is ` G ` in this context? Is this hypothesis really necessary?

Reply:
Indeed I guess here all that's needed is that `.0.` is a set, like in `.0. e. W`.